scripts/yul_coverage.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")"/..
+
+for arg in "$@"; do
+  case "$arg" in
+  --no-stats) NO_STATS=1 ;;
+  --successful) SHOW_SUCCESSFUL=1 ;;
+  --internal-compiler-errors) SHOW_UNIMPLEMENTED_FEATURE_ERRORS=1 ;;
+  --unimplemented-feature-errors) SHOW_INTERNAL_COMPILER_ERRORS=1 ;;
```

Comment:
Is this intentionally reversed :-)?

Reply:
No. Just a bug in my suggestion above. Sorry, I wasn't careful enough.

Reply:
Oh, actually it was there before my suggestion.
Anyway, I'm pretty sure it's just a bug. It should be swapped.

Reply:
Ups